src/org.ts

```diff
@@ -640,6 +654,69 @@ export class Org extends AsyncCreatable<Org.Options> {
     throw new SfdxError('Not Supported');
   }
 
+  /**
+   * Query the sandbox for the SandboxProcessObject by sandbox name
+   *
+   * @param sandboxName The name of the sandbox to query
+   * @returns {SandboxProcessObject} The SandboxProcessObject for the sandbox
+   */
+  private async queryLatestSandboxProcessBySandboxName(sandboxNameIn: string): Promise<SandboxProcessObject> {
+    const { tooling } = this.getConnection();
+    const SOBJECT_SANDBOXPROCESS = 'SandboxProcess';
+    const QUERY_SANDBOXPROCESS_FIELDS =
```

Comment:
can these vars be injected into `queryStr` directly? They aren't referenced anywhere else